python-packages/sra_client/test/relayer/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '3'
 services:
     ganache:
-        image: "0xorg/ganache-cli:2.2.2"
+        image: "0xorg/ganache-cli:4.4.0-beta.1"
```

Comment:
Was this intentional?

Reply:
Yes, to stay in sync with the `.circleci/config.yml`